piece.gd

```diff
@@ -49,8 +52,12 @@ func initialize(spawn_position: Vector2, piece_data: Dictionary):
 func _process(delta):
     lock_time += delta
     current_input_delay += delta
+    current_key_held_time += delta
     board.Board_clearPiece(self)
-    if current_input_delay > input_delay:
+    if (
+        current_input_delay > standard_movement_speed
+        || (current_key_held_time > lag_delay && current_input_delay > hold_movement_speed)
+    ):
```

Comment:
Can you add a few comments explaining how the acceleration works? Not intuitive from a glance.